### PR TITLE
Further erpc work

### PR DESCRIPTION
**Pull request by danderson-presonus**
This just to merge in the changes I had to do to get rpmsg-lite working on the imx8m nano.

### CODE REVIEW

File: lib/include/platform/imx8mn_m7/rpmsg_platform.h [hunk 1]
```diff
@@ -36,6 +37,20 @@
 #define RL_PLATFORM_IMX8MN_M7_USER_LINK_ID (0U)
 #define RL_PLATFORM_HIGHEST_LINK_ID        (15U)
 
+
+/* 
+	Shared memory base for RPMsg communication. 
+	TODO: put me somewhere sensible
+*/
+#define VDEV0_VRING_BASE	  (0xB8000000U)
```
Comment (baudiffred) [inline lib/include/platform/imx8mn_m7/rpmsg_platform.h:45]: Maybe pass from compiler args so non-default values come from higher level?

**Review by baudiffred — APPROVED** [review]

File: lib/include/platform/imx8mn_m7/rpmsg_platform.h (hunk 1 above)
Comment (danderson-presonus) [reply lib/include/platform/imx8mn_m7/rpmsg_platform.h:45]: similar to this, but in pure C?
namespace Pep {

namespace Devices {

	//!< max total devices
	#ifdef CONFIG_DEVICE_MAX_DEVICES
	  constexpr int kMaxDevices = CONFIG_DEVICE_MAX_DEVICES;	
	#else
	  constexpr int kMaxDevices = 10;	
	#endif

Comment (baudiffred) [reply lib/include/platform/imx8mn_m7/rpmsg_platform.h:45]: Not even that fancy. 
```
#ifndef VDEV0_VRING_BASE
#define VDEV0_VRING_BASE
#endif
```
Pass override value in cmake if needed.

Comment (danderson-presonus) [reply lib/include/platform/imx8mn_m7/rpmsg_platform.h:42]: done

**Pull request merged**